pkg/liveshare/session.go

```diff
@@ -123,6 +123,20 @@ func (s *Session) StartJupyterServer(ctx context.Context) (int, string, error) {
 	return port, response.ServerUrl, nil
 }
 
+func (s *Session) Rebuild(ctx context.Context) error {
+	var rebuildSuccess bool
+	err := s.rpc.do(ctx, "IEnvironmentConfigurationService.rebuildContainer", []string{}, &rebuildSuccess)
+	if err != nil {
+		return fmt.Errorf("invoking rebuild RPC: %w", err)
+	}
+
+	if !rebuildSuccess {
+		return fmt.Errorf("couldn't rebuild codespace")
```

Comment:
Is there a log file or any more descriptive message we could provide or point to?

Reply:
I don't think so. The RPC call _only_ returns `true`/`false`. Unfortunately, there doesn't seem to be a more detailed message available here.

Note that the lines above this one check for an error after making the RPC call. If something goes wrong there, we get a helpful error message.

Reply:
We could modify the RPC service to return a more helpful error message. That's how the Jupyter and SSH RPC calls work: https://github.com/cli/cli/blob/a493f6dbe888b43625aaa8b2c7e01ca6445da1b2/pkg/liveshare/session.go#L105